docs/getting-started/windows-manual.md

```diff
@@ -72,3 +84,18 @@ Start-Service splunk-otel-collector
 ```
 
 The collector logs and errors can be viewed in the Windows Event Viewer.
+
+### Docker
+
+Create build of otel collector on local system.
+```bash
+$ git clone https://github.com/signalfx/splunk-otel-collector.git
+$ cd splunk-otel-collector
+$ docker build -t otelcol --build-arg SMART_AGENT_RELEASE=5.11.2 -f .\cmd\otelcol\Dockerfile.windows .\cmd\otelcol\
+```
+
+Deploy the latest Docker image.
+
+```bash
+$ docker run --rm -e SPLUNK_ACCESS_TOKEN=12345 -e SPLUNK_REALM=us0 -p 13133:13133 -p 14250:14250 -p 14268:14268 -p 4317:4317 -p 6060:6060 -p 8888:8888 -p 9080:9080 -p 9411:9411 -p 9943:9943 --name=otelcol otelcol:latest
```

Comment:
We'll need to decide what to tag/push released images as.  Maybe `quay.io/signalfx/splunk-otel-collector-windows:<VERSION>` or `quay.io/signalfx/splunk-otel-collector:<VERSION>-windows` or something else?  For reference, the smart agent is `quay.io/signalfx/signalfx-agent:<VERSION>-windows` and there is no `latest` tag.

We'll also need to add some documentation for how to run the image with the `SPLUNK_CONFIG` env var for a custom or volume mounted config file.

Reply:
Do I need to make any change in the codebase regarding tag/push image in quay.io? also I have updated the documentation for running image with `SPLUNK_CONFIG` env var.

Reply:
I want to ask that to push windows image on `quay.io` do I need to change https://github.com/signalfx/splunk-otel-collector/blob/main/.gitlab-ci.yml? or this will not be part of this PR?

Reply:
> I want to ask that to push windows image on `quay.io` do I need to change https://github.com/signalfx/splunk-otel-collector/blob/main/.gitlab-ci.yml? or this will not be part of this PR?

Our gitlab pipeline currently doesn't support windows, so we'll need to figure that out along with what to tag the image.